bin/did-calypso-app-change.mjs

```diff
@@ -57,7 +57,7 @@ async function downloadPrevBuild( appSlug, dir ) {
 	const { body, status } = await fetch( prevBuildUrl );
 	if ( status !== 200 ) {
 		throw new Error(
-			`Could not fetch previous build for ${ appSlug }! Response code ${ status }.`
+			`Could not fetch previous build for ${ appSlug }! Response code ${ status }. You can ignore this error when first adding a new app — just add a \`teamcity:build-app\` package script so it's ready for subsequent PRs.`
```

Comment:
I think we also need to add the new release tag to this allowlist:

https://github.com/Automattic/wp-calypso/blob/c0a10a96671daa6a1e4c15279dea0079e56157f2/.teamcity/_self/projects/WPComPlugins.kt#L38-L45

Likely something that's also worth documenting.

Reply:
Resolving in #103118

Reply:
#103127 actually. Closed mine in favor of yours.